src/main/java/frc/robot/commands/intake/floor/FloorDrop.java

```diff
@@ -9,21 +9,32 @@
 
 public class FloorDrop extends CommandBase {
   /** Creates a new FloorIntakeOut. */
-  FloorIntake floorIntake;
+  private FloorIntake floorIntake;
+  private boolean fast = true;
   public FloorDrop(FloorIntake floorIntake) {
     // Use addRequirements() here to declare subsystem dependencies.
     this.floorIntake = floorIntake;
     addRequirements(this.floorIntake);
   }
 
+  public FloorDrop(FloorIntake floorIntake, boolean fast) {
```

Comment:
I think this would be most clear if we pass in the speed as double value to the constructor. This is nice because our button bindings become very readable.

new FloorDrop(floorIntake, Constants.FLOOR_INTAKE_NORMAL_SPEED)
new FloorDrop(floorIntake, Constants.FLOOR_INTAKE_FAST_SPEED)